### DISCUSSION TITLE
France 4

### DISCUSSION THREAD

**Discussion by eltoncomp** [iptv-org/community#730] [Q&A]
**Channel Name:** France 4
**Country:** France
**Language:** French
**Website:** https://mobile.france.tv/france-4
**Stream URL (optional):** xxx
**Notes (optional):** All France Televisions channels are in the playlist, only France 4 is missing

**Comment by dct-infra** [comment]
Stream out of https://mobile.france.tv/france-4 is geo blocked and IP locking.
Not much to do about this until an open free stream shows up somewhere

**Comment by eltoncomp** [comment]
http://edge9.iptvnetwork.net/live/ is the server where France 2, 3 and 5 are hosted.
The PC version is https://www.france.tv/france-4/
I posted the mobile version because I looked for the France 4 website in a smartphone.

**Comment by PopeyeTheSai10r** [comment]
![Capture](https://github.com/iptv-org/iptv/assets/107763146/af2700f4-8784-4650-9027-607800ee442d)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/fr.m3u#L72-L73
